app/models/service_ansible_playbook.rb

```diff
@@ -171,6 +171,11 @@ def decrypt_options(opts)
 
   def log_stdout(action)
     log_option = options.fetch_path(:config_info, action.downcase.to_sym, :log_output) || 'on_error'
-    playbook_log_stdout(log_option, job(action))
+    job = job(action)
+    if job.nil?
+      $log.info("No stdout available due to missing job")
```

Comment:
@tinaafitz Is this log message sufficient?  I just threw in some text, but wasn't sure this was enough (like should there be an expect job id in the message itself for log correlation or anything?)

Reply:
Hi @Fryguy Yes. The log message is sufficient. Thanks. :-)